.idea/checkstyle-idea.xml

```diff
@@ -3,14 +3,12 @@
   <component name="CheckStyle-IDEA">
     <option name="configuration">
       <map>
-        <entry key="active-configuration" value="LOCAL_FILE:$PROJECT_DIR$/config/checkstyle.xml:a8c Style" />
-        <entry key="checkstyle-version" value="8.2" />
+        <entry key="checkstyle-version" value="8.20" />
         <entry key="copy-libs" value="false" />
         <entry key="location-0" value="BUNDLED:(bundled):Sun Checks" />
         <entry key="location-1" value="BUNDLED:(bundled):Google Checks" />
-        <entry key="location-2" value="LOCAL_FILE:$PROJECT_DIR$/config/checkstyle.xml:a8c Style" />
         <entry key="scan-before-checkin" value="false" />
-        <entry key="scanscope" value="AllSourcesWithTests" />
+        <entry key="scanscope" value="JavaOnly" />
```

Comment:
Why are you making changes to the `.idea` files? This was probably done automatically by your IDE, you'll need to undo this.